style.css

```
@@ -0,0 +1,448 @@
+    sans-serif;
+    background-color: palegreen;
+    max-width: 900px;
+    margin: auto;
+}
+
+hr {
+    margin: 2rem 0rem;
+}
+
+
+img {
+height: 300px;
+background-position: center;
+background-repeat: no-repeat;
+background-size: cover;
+margin: 10px 10px;
+}
+
+
+.welcome {
+    text-align: center;
+}
+
+.avatar-container {
+    display: inline-block;
+    position: relative;
+}
+
+.avatar {
+    border-radius: 50%;
+    object-fit: cover;
+}
+
+.avatar--s {
+width: 5rem;
+height: 5rem;
+}
+
+.avatar--m {
+width: 10rem;
+height: 10rem;
+}
+
+.avatar--l {
+width: 15rem;
+height: 15rem;
+}
+
+.alert{
+width:50%;
+margin:20px auto;
+padding:30px;
+position:relative;
+border-radius:5px;
+box-shadow:0 0 15px 5px #ccc;
+background-color:white;
+
+}
+
+.warning-alert{
+    width:50%;
+    margin:20px auto;
+    padding:30px;
+    position:relative;
+    border-radius:5px;
+    box-shadow:0 0 15px 5px #ccc;
+    background-color:yellow;
+    
+}
+
+.danger-alert{
+    width:50%;
+    margin:20px auto;
+    padding:30px;
+    position:relative;
+    border-radius:5px;
+    box-shadow:0 0 15px 5px #ccc;
+    background-color:red;
+    
+}
+
+.success-alert{
+    width:50%;
+    margin:20px auto;
+    padding:30px;
+    position:relative;
+    border-radius:5px;
+    box-shadow:0 0 15px 5px #ccc;
+    background-color:green;
+    
+}
+
+.badge {
+    position: absolute;
+}
+  
+.badge--round {
+    border-radius: 50%;
+
+    height: 15px;
+    width: 15px;
+}
+
+.badge-border {
+    border: 2px solid white;
+}
+
+.badge--top {
+    top: 10px;
+    right: 10px;
+}
+
+.badge-active {
+    background-color: var(--active);
+}
+
+.badge-inactive {
+    background-color: var(--inactive);
+}
+
+.btn {
+    padding: 0.9rem 1.3rem;
+    border-radius: 0.8rem;
+}
+
+.btn-primary {
+    background-color: blue;
+}
+
+.btn-secondary {
+    background-color: gray;
+}
+
+.btn-success {
+    background-color: green;
+}
+
+.btn-danger {
+    background-color: red;
+}
+
+a:link {
+    background-color: gray;
+    color: white;
+    padding: 10px 15px;
+    text-align: center;
+    display: inline-block;
+}
+
+.btnfloat {
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  padding: 0.5rem;
+}
+
+.btnfloat .btn-primary {
+  font-size: 2rem;
+}
+
+.icon-badge-group .icon-badge-container {
+    display: inline-block;
+    margin-left: 15px;
+}
+
+.icon-badge-container {
+    margin-top: 20px;
+    position: relative;
+    display: inline-block;
+}
+
+.icon-badge-icon {
+  font-size: 30px;
+  position: relative;
+}
+
+.notification {
+    position: relative;
+}
+  
+.notification-icon {
+    font-size: 4rem;
+}
+  
+.list-non-bullet {
+    list-style: none;
+}
+
+.list-item-inline-components {
+    display: inline;
+    padding: 0rem 0.5rem;
+    justify-content: left;
+    color: black;
+}
+
+.list-item-inline-github {
+    display: inline;
+    padding: 0rem 0.5rem;
+    justify-content: right;
+    color: black;
+}
+
```

Changes for line 198 to 210
```suggestion
.list-item-inline-components, .list-item-inline-github {
    display: inline;
    padding: 0rem 0.5rem;
    justify-content: left;
    color: black;
}

.list-item-inline-github {
    justify-content: right;
}
```